lib/dal/src/workspace_snapshot.rs

```diff
@@ -843,39 +855,12 @@ impl WorkspaceSnapshot {
             .to_owned())
     }
 
-    pub async fn get_node_weight(
-        &self,
-        node_index: NodeIndex,
-    ) -> WorkspaceSnapshotResult<NodeWeight> {
-        Ok(self
-            .working_copy()
-            .await
-            .get_node_weight(node_index)?
-            .to_owned())
-    }
-
-    pub async fn get_node_weight_opt(&self, node_index: NodeIndex) -> Option<NodeWeight> {
-        self.working_copy()
-            .await
-            .get_node_weight_opt(node_index)
-            .map(ToOwned::to_owned)
-    }
-
-    #[instrument(
-        name = "workspace_snapshot.find_equivalent_node",
-        level = "debug",
-        skip_all,
-        fields()
-    )]
-    pub async fn find_equivalent_node(
-        &self,
-        id: Ulid,
-        lineage_id: Ulid,
-    ) -> WorkspaceSnapshotResult<Option<NodeIndex>> {
-        Ok(self
-            .working_copy()
-            .await
-            .find_equivalent_node(id, lineage_id)?)
+    pub async fn get_node_weight_opt(&self, id: impl Into<Ulid>) -> Option<NodeWeight> {
+        let working_copy = self.working_copy().await;
+        working_copy
+            .get_node_index_by_id_opt(id)
+            .and_then(|node_index| working_copy.get_node_weight_opt(node_index))
+            .cloned()
     }
```

Comment:
This method is never used (the graph version is used internally by the internal graph, but it's never used by external consumers of the snapshot)